### PR TITLE
update operand/operator images version

### PR DESCRIPTION
**Pull request by siddarthpatel**
Still need to figure out the versioning with the channels in our package.yaml
There is an ongoing discussion in slack..
Do not merge

### CODE REVIEW

File: pkg/resources/container.go [hunk 1]
```diff
@@ -28,11 +28,11 @@ import (
 
 const DefaultImageRegistry = "quay.io/opencloudio"
 const DefaultImageName = "common-web-ui"
-const DefaultImageTag = "1.1.0"
+const DefaultImageTag = "1.2.0"
```
Comment (ericabr) [inline pkg/resources/container.go:31]: need to change the image tag of our operator too

**Comment by siddarthpatel** [comment]
Discussed this PR with jiaming, he says it looks correct. 
Here is their PR for updating the operator version for reference https://github.com/IBM/operand-deployment-lifecycle-manager/pull/287/files

**Comment by siddarthpatel** [comment]
@ericabr Reverted `operand images version`. Keeping the updated `operator version` though.

File: deploy/olm-catalog/ibm-commonui-operator/1.2.0/ibm-commonui-operator.v1.2.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+          "kind": "CommonWebUI",
+          "metadata": {
+            "name": "example-commonwebui"
+          },
+          "spec": {
+            "commonWebUIConfig": {
+              "cpuLimits": "300m",
+              "cpuMemory": "200Mi",
+              "imageRegistry": "quay.io/opencloudio",
+              "imageTag": "1.2.0",
```
Comment (ericabr) [inline deploy/olm-catalog/ibm-commonui-operator/1.2.0/ibm-commonui-operator.v1.2.0.clusterserviceversion.yaml:18]: shouldn't this be 1.1.0?

File: deploy/olm-catalog/ibm-commonui-operator/1.2.0/ibm-commonui-operator.v1.2.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+          "kind": "LegacyHeader",
+          "metadata": {
+            "name": "example-legacyheaderservice"
+          },
+          "spec": {
+            "legacyConfig": {
+              "cpuLimits": "300m",
+              "cpuMemory": "200Mi",
+              "imageRegistry": "quay.io/opencloudio",
+              "imageTag": "3.2.5",
```
Comment (ericabr) [inline deploy/olm-catalog/ibm-commonui-operator/1.2.0/ibm-commonui-operator.v1.2.0.clusterserviceversion.yaml:52]: 3.2.4?

**Comment by siddarthpatel** [comment]
@ericabr You're right. Updated the csv

**Pull request merged**